apps/fluent-tester/src/FluentTester/TestComponents/CheckboxExperimental/CheckboxTest.tsx

```diff
@@ -3,11 +3,12 @@ import { EXPERIMENTAL_CHECKBOX_TESTPAGE } from './consts';
 import { Test, TestSection, PlatformStatus } from '../Test';
 import { Checkbox } from '@fluentui-react-native/experimental-checkbox';
 import { useTheme } from '@fluentui-react-native/theme-types';
-import { View, TextInput, TextStyle } from 'react-native';
+import { View, TextInput, TextStyle, AccessibilityActionEvent, GestureResponderEvent } from 'react-native';
 import { commonTestStyles as commonStyles } from '../Common/styles';
 import { E2ECheckboxExperimentalTest } from './E2ECheckboxExperimentalTest';
+import { KeyPressEvent } from '@fluentui-react-native/interactive-hooks';
 
-function onChangeUncontrolled(isChecked: boolean) {
+function onChangeUncontrolled(_e: GestureResponderEvent | KeyPressEvent | AccessibilityActionEvent, isChecked: boolean) {
```

Comment:
Do we want to define our own type for this intersection? Or leave it explicit and let React Native Core/windows/macOS deal with creating a type we can later use?

Reply:
I think leaving it explicit is better for clients to understand what kind of event info comes through, but copy/pasting this everywhere is pretty cumbersome... so yeah I think defining our own union type would be good. Let me figure out where that should live...

Reply:
I would imagine interactive hooks, but 🤷🏾‍♂️

Reply:
Done in latest commits